### ISSUE TITLE
`create-vue-application` fails when used with a standalone plugin

### ISSUE THREAD

**Issue by chrabyrd** [archesproject/arches#10984]
As of #10976 , `create-vue-application` fails because it's using `arches.urls` to fetch its i18n data, but `javascript.htm` -- which is used to stock the data in the `arches` component, is not used.

### STEPS TO REPRODUCE:

1. Edit this line in `js/utils/create-vue-application.js`:
    from:
    ```
    return fetch(arches.urls.api_get_frontend_i18n_data).then(function(resp) {
    ```

    to: 
    ```
    return fetch('/api/get_frontend_i18n_data').then(function(resp) {
    ```

2. Create a standalone plugin. Go through the process of creating a plugin but ensure the `config` property has 
    ```
    "is_standalone": true
    ```
    as a k/v pair, and ensure `component` is pointing to a Vue component, eg:

    ```
    src/ExampleComponent.vue
    ```

3. Visit the plugin. It will not be listed in the sidebar, so you will need to manually navigate to the plugin. eg `localhost:8000/plugins/my-plugin`

4. Notice the component loads successfully.

5. Now revert this line in `js/utils/create-vue-application.js`:
    from:
    ```
    return fetch('/api/get_frontend_i18n_data').then(function(resp) {
    ```

    to: 
    ```
    return fetch(arches.urls.api_get_frontend_i18n_data).then(function(resp) {
    ```

6. Notice the component fails to load.

### PROPOSED SOLUTION

- Move the `arches_urls` block outside of `javascript.htm`
- `include` that template in `javascript.htm` and `plugin-standalone.htm`

This will allow developers to access arches from standalone plugins while avoiding duplicating the urls themselves

**Comment by chrabyrd** [comment]
potentially resolved via removing the feature entirely in https://github.com/archesproject/arches/issues/11001 -- however the proposed solution in this ticket will be implemented

**Comment by chrabyrd** [comment]
resolved  in #11002
